install-mir-json.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+TOOLCHAIN="nightly-2023-01-23"
+
+# Check for rustup
+if ! command -v rustup >/dev/null 2>&1; then
+  echo "rustup is not installed."
+  echo "Please install rustup by running:"
+  echo "  curl https://sh.rustup.rs -sSf | sh"
+  echo "and update your path:"
+  echo "  source $HOME/.cargo/env"
```

Comment:
Instead of putting the exact shell commands to run here, I'd recommend pointing folks to the instructions at https://rustup.rs instead, as it actually recommends running a different command than what is listed here. What's more, this would future-proof the script in the event that the https://rustup.rs instructions change later.